OJP/OJP_Trips.xsd

```diff
@@ -396,7 +396,7 @@
 			<xs:documentation>Structure for trip overview information.</xs:documentation>
 		</xs:annotation>
 		<xs:sequence>
-			<xs:element name="TripId" type="ObjectIdType">
+			<xs:element name="TripId" type="xs:NMTOKEN">
```

Comment:
I think this breaks the last update?

Reply:
I think this will be weeded out during the rebasing.... But I change it here as well.